public/app/features/alerting/unified/api/alertRuleApi.ts

```diff
@@ -413,5 +418,11 @@ export const alertRuleApi = alertingApi.injectEndpoints({
       }),
       keepUnusedDataFor: 0,
     }),
+    getDeletedRules: build.query<RulerGrafanaRulesConfigDTO, {}>({
+      query: () => ({
+        url: `/api/ruler/${GRAFANA_RULES_SOURCE_NAME}/api/v1/rules/`,
+        params: { deleted: 'true' },
+      }),
+    }),
```

Comment:
We need to add a tag to this endpoint, and invalidate it when we delete a rule - otherwise when you delete a rule and go straight to the new page, you don't see it